Hymns/Display/HymnLyricsView.swift

```diff
@@ -3,7 +3,8 @@ import Resolver
 
 public struct HymnLyricsView: View {
 
-    @ObservedObject private var viewModel: HymnLyricsViewModel
+    //Pass in from DetailHymnScreen
```

Comment:
I'm unclear why this is no longer observed?  Is it because there's no binding on the lyrics?

If that's the case, can we also remove the @Published on the view model as well

Reply:
Yeah, I don't think it needs to be observed at this point since there is nothing binding and it should never change within the view. If there is any change to this later we can make it observed again. And yeah, I think we can also remove the @published.